src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/slideshow/SlideshowBuilderComponentObject.java

```diff
@@ -28,32 +28,17 @@ public SlideshowBuilderComponentObject(WebDriver driver) {
   }
 
   public void adjustWidth(int width) {
-    //TODO
+    // TODO
   }
 
   public void useSmartCropping() {
-    //TODO
-  }
-
-  public enum Positions {
-    LEFT, CENTER, RIGHT;
-
-    private final String label;
-
-    Positions() {
-      this.label = StringUtils.capitalize(this.toString().toLowerCase());
-    }
-
-    public String getPosition() {
-      return this.label;
-    }
+    // TODO
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)